recipes-asteroid/asteroid-launcher/asteroid-launcher/jolla-supported-languages/de_DE.conf

```diff
@@ -0,0 +1,4 @@
+Name=Deutsch
+LocaleCode=de_DE.utf8
+Region=Deutschland
+RegionLabel=Region: %1
```

Comment:
Let's create a new supported-language recipe and get rid of the jolla prefix in nemo-qml-plugin-systemsettings